spec/models/basket_spec.rb

```diff
@@ -6,16 +6,20 @@
   describe '#add_product' do
     subject { basket.add_product(product_id) }
 
+    let(:items) do
+      double(
+        'ActiveRecord::Relation',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.